bpf/Makefile

```diff
@@ -265,6 +270,11 @@ MAX_LXC_OPTIONS = $(MAX_BASE_OPTIONS) -DENCAP_IFINDEX=1 -DTUNNEL_MODE=1
 ifneq (,$(filter $(KERNEL),"54" "netnext"))
 MAX_LXC_OPTIONS += -DENABLE_EGRESS_GATEWAY=1
 endif
+ifeq ("$(KERNEL)","54")
+MAX_LXC_OPTIONS+= -DENABLE_VTEP=1
+else ifeq ("$(KERNEL)","netnext")
+MAX_LXC_OPTIONS+= -DENABLE_VTEP=1
```

Comment:
Have you tested this?

At a glance, it looks like with the PR in the current state, if you run `KERNEL=54 make -C bpf/` or similar (in order to enable one of these lines), then it will enable the code but `VTEP_NUMS`, `VTEP_ENDPOINT`, `VTEP_MAC` will be undefined. These likely need to be configured somewhere in the `bpf/*.h` files for testing purposes. They should be defined in a way that tests the maximum expected complexity that any user could configure, ie the highest number of VTEPs that are supported. This way, the automated testing in CI will build this combination of features and will tell us whether this feature will cause problems for users at runtime.

Reply:
good point, I did not know to test `KERNEL=54 make -C bpf/`, I can add something like below in 

```
diff --git a/bpf/node_config.h b/bpf/node_config.h
index cdf66edd4a..5fddf67d2e 100644
--- a/bpf/node_config.h
+++ b/bpf/node_config.h
@@ -176,6 +176,12 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
 # define LB_SELECTION          LB_SELECTION_RANDOM
 #endif
 
+#ifdef ENABLE_VTEP
+#define VTEP_ENDPOINT (__u32[]){0xec48a90a, 0xee48a90a, }
+#define VTEP_MAC (__u64[]){0x562e984c3682, 0x552e984c3682, }
+#define VTEP_NUMS 4
+#endif
+

```

tested `KERNEL=54 make -C bpf/`, compiles ok, in regard to number of VTEPs we want to limit, I am thinking maybe 4 maximum, and  I could add code in the `validateVTEP()` in `pkg/option/config.go` to check the VTEP number limit. what do you think?

Reply:
That seems acceptable to me.